src/sentry/static/sentry/app/components/events/meta/annotatedText/index.tsx

```diff
@@ -88,13 +92,14 @@ const StyledTooltipError = styled(Tooltip)`
   vertical-align: middle;
 `;
 
-const StyledListItem = styled(ListItem)`
-  padding-left: ${space(3)};
-`;
-
 const StyledList = styled(List)`
-  li:before {
-    border-color: ${p => p.theme.white};
+  li {
+    padding-left: ${space(3)};
+    :before {
+      border-color: ${p => p.theme.white};
+      top: 50%;
+      transform: translateY(-50%);
```

Comment:
_nit_: Can we just use regular bullet point styling here? Basically a filled bullet aligned with the first line of text?

Reply:
. This is the bullet point that we currently use and I would have to ask the design if we can introduce a new one.
. the symbol alignment I can do  as you suggested 😉